httpclient5/src/main/java/org/apache/hc/client5/http/impl/async/HttpAsyncClientBuilder.java

```diff
@@ -318,7 +320,15 @@ public final HttpAsyncClientBuilder setIOReactorConfig(final IOReactorConfig ioR
         this.ioReactorConfig = ioReactorConfig;
         return this;
     }
-
```

Comment:
@cda007 Could you please add `@since 5.2` tag?

Reply:
done